### DISCUSSION TITLE
Add Automatic Cursor Size Scaling with Circle Size to Lazer

### DISCUSSION THREAD

**Discussion by OneShotFox** [ppy/osu#29308] [Ideas]
I just play with this setting turned on in stable and been playing the daily challenges. I was just wondering if it could get added to stable for consistency sake :>
![image](https://github.com/user-attachments/assets/e35b0098-6bf6-499d-86d4-611b4e9a48b6)

Ignore this i literally just found the option after posting these after not finding it for 30 minutes

**Comment by peppy** [comment]
Were you searching for a specific term that meant you couldn't find it?
